frontend/src/Movie/Delete/DeleteMovieModalContent.js

```diff
@@ -108,23 +108,23 @@ class DeleteMovieModalContent extends Component {
           {
             deleteFiles &&
               <div className={styles.deleteFilesMessage}>
-                <div>The movie folder <strong>{path}</strong> and all it's content will be deleted.</div>
+                <div>{translate('DeleteTheMovieFolder', { path })}</div>
```

Comment:
This was marked as resolved but it still isn't square brackets

Reply:
now its good